src/components/Advertising.tsx

```diff
@@ -80,7 +80,7 @@ const Advertising = () => {
             mb={4}
             justifyContent={["left", "left", "left", "center"]}
           >
-            <Img src={salazraki} alt="salazraki advert" mb={4}></Img>
+            <Img src={salazraki} alt="Salazraki advert" mb={4}></Img>
```

Comment:
alt tags are supposed to be a text replacement for when images dont load or they get read out to someone using a screen reader. So the alt tags need to be something more like `Salazraki icon`

Reply:
the same type of change to be applied to all other alt tags please :)

Reply:
ah okay.. that's good to know I wasn't sure from reading accessibility blogs if I had to actually put what was going on in the picture because if so then _lol_ at all the interesting alt tags there'd be in this code...